app/lib/presentation/organisms/select_language_card_organism.dart

```diff
@@ -0,0 +1,95 @@
+import 'package:flutter/material.dart';
+import 'package:word_link/domain/controllers/controllers.dart';
+import 'package:word_link/presentation/core/global_variables.dart';
+
+class SelectLanguageCardOrganism extends StatelessWidget {
+  const SelectLanguageCardOrganism({
+    required this.language,
+    required this.isSelected,
+    required this.onTap,
+    super.key,
+  });
+
+  final LanguageEnum language;
+  final bool isSelected;
+  final VoidCallback onTap;
+
+  @override
+  Widget build(BuildContext context) {
+    return GestureDetector(
+      onTap: onTap,
+      child: Container(
+        decoration: BoxDecoration(
+          borderRadius: BorderRadius.circular(GlobalVariables.defaultRadius),
+          boxShadow: [
+            BoxShadow(
+              color: Theme.of(context).colorScheme.shadow,
+              blurRadius: 10,
+              offset: const Offset(0, 5),
+            ),
+          ],
+          border: isSelected
+              ? Border.all(
+                  color: Theme.of(context).colorScheme.primary, width: 3)
+              : null,
+        ),
```

Comment:
The number should come from the theme.
You can add BoxShadow to theme or the values and use them